src/pages/guideline_ios/index.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import Flex from '../../components/Flex';
+import Layout from '../../components/Layout';
+import LeftNav from '../../components/LeftNav';
+import designImg from './../../img/design.png';
+import { Link, graphql } from 'gatsby';
+import ResourcesList from '../../components/ResourceList/ResourcesList';
+import Tabs from '../../components/Tabs';
+import { css } from '@emotion/core';
+
+export default class GuidelineIosIndexPage extends React.Component {
+  render() {
+    const { data, location } = this.props;
+    const frontmatter = data.ios.edges[0].node.frontmatter;
+    const posts = data.posts.edges;
+
+    return (
+      <Layout location={location}>
+        <Flex
+          direction='row'
+          shrink='0'
+          grow='1'
+          overflow='auto'
+          valign='stretch'
+          css={{
+            width: '100%',
+          }}
+        >
+          <LeftNav title="Fiori For Web" open="false" data={data.leftNav} />
+          <div
+            css={{
+              width: '100%'
+            }}
+          >
+            <div
+              css={{
+                width: '100%',
+                backgroundColor: '#f8f9fb',
+                height: '682px',
+                backgroundImage: 'url(' + designImg + ')',
+                backgroundRepeat: 'no-repeat',
+                backgroundPosition: 'top right'
+              }}
+            />
+            <div css={{
+              width: 828,
+              margin: '0 auto',
+              paddingBottom: 120,
+              paddingTop: 120,
+              transition: 'width 0.3s ease-in-out'
+            }}>
+              <h1 css={css`
+              color: #4F4F4F;
+              font-family: 72-Regular;
+              font-size: 36px;
+              font-weight: normal;
+              height: 43px;
+              letter-spacing: 0.11px;
+              line-height: 43px;`}>explore Fiori for iOS</h1>
+              <Tabs>
+                {frontmatter.tabs.map((tab, idx) => {
+                  return (
+                    <div label={tab} key={idx}>
+                      Hello from {tab}!
```

Comment:
is this final?

Reply:
nope, we are going to have some cards there